pyServer/manifests/linux/normal

```diff
@@ -20,3 +20,4 @@ copy,/var/log/cloud-init*
 copy,/var/log/boot*
 copy,/var/log/auth*
 copy,/var/log/secure*
+copy,/var/log/xrdp*,noscan
```

Comment:
Can you provide below details for this?
-- How many files we are collecting here as '*' added at the end
-- What will be average of total size of all files here?
-- Can these logs contain any secrets? As you added "noscan" at the end it must not contain any secrets.

And please follow below steps to create pull request in this repo, as it does not allow to have pull request from forked.

Reply:
- How many files we are collecting  ? 2  (xrdp.log, xrdp-sesman.log)
- What will be average of total size of all files here? 2 MB
- Can these logs contain any secrets? no

Which step do I need ? Looks like Travis CI is failing because it could not download Python 3.5, can you help ?

Reply:
Missed adding link for steps:
https://github.com/Azure/azure-diskinspect-service#how-to-contribute

It's not about python downloading issue, actually it's blocking build because not generated from current repo.